atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Leaders.java

```diff
@@ -155,9 +156,6 @@ public static LocalPaxosServices createInstrumentedLocalServices(
                 PTExecutors.newCachedThreadPool(daemonThreadFactory("atlas-proposer")),
                 metricsManager.getRegistry(),
                 MetricRegistry.name(PaxosProposer.class, "executor"));
-        PaxosProposer proposer = AtlasDbMetrics.instrument(metricsManager.getRegistry(), PaxosProposer.class,
```

Comment:
It looks like this change drops the `PaxosProposer` metrics, which I'm not sure we want to do.

Reply:
makes things a bit awkward maybe, as now I have to pass in a metrics registry, and you might want to instrument it with certain tags/metrics etc. Another thought I had was having some sort of decorator function which is just a `UnaryOperator` on the builder which would allow you to instrument the metrics as you see fit, but seemed a bit overkill, but can do